google/services/bigquery/resource_bigquery_table.go

```diff
@@ -1982,7 +1982,11 @@ func resourceBigQueryTableColumnDrop(config *transport_tpg.Config, userAgent str
 	}
 
 	if len(droppedColumns) > 0 {
-		droppedColumnsString := strings.Join(droppedColumns, ", DROP COLUMN ")
+		backquotedDroppedColumns := []string{}
+		for _, column := range droppedColumns {
+			backquotedDroppedColumns = append(backquotedDroppedColumns, fmt.Sprintf("`%s`", column))
+		}
+		droppedColumnsString := strings.Join(backquotedDroppedColumns, ", DROP COLUMN ")
```

Comment:
If `droppedColumns = []string("column1", "column2")` ,

backquotedDroppedColumns = []string("`column1`", "`column2`")
droppedColumnsString = "`column1`, DROP COLUMN `column2`"
dropColumnsDDL = "ALTER TABLE <table_id> DROP Column `column1`, DROP COLUMN `column2`"

Reply:
This PR should be opened on the `magic-modules` repo instead as it's the code that generates the `terraform-provider-google` repo.

The exact file you would edit can be found here: https://github.com/GoogleCloudPlatform/magic-modules/blob/37e4abb059553f92e3eb4960ec0a2188a3cef77a/mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.tmpl#L2086-L2103

I'll be closing this PR, please make the changes in `magic-modules`!